CHANGELOG-17.0.md

```diff
@@ -36,4 +36,373 @@ You can see all the changes done to the package that you carry about with this t
+- upgrade Elasticsearch version to 7.17.2 by @henzigo in https://github.com/shopsys/shopsys/pull/3874
+- fix cypress screenshots by @grossmannmartin in https://github.com/shopsys/shopsys/pull/3888
+- db migrations: add empty down() method into AbstractMigration by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/3887
+- Upgrade Deployment package by @henzigo in https://github.com/shopsys/shopsys/pull/3891
+- determining file upload type: use instanceof because of entity overriding by @thirdknown in https://github.com/shopsys/shopsys/pull/3766
+- make uniqid more unique by @henzigo in https://github.com/shopsys/shopsys/pull/3906
+- Update storefront dependencies by @JanMolcik in https://github.com/shopsys/shopsys/pull/3899
+- makefile improvements + removed redundant robots.txt data fixtures by @TomasLudvik in https://github.com/shopsys/shopsys/pull/3910
+- Create crud skeleton by @henzigo in https://github.com/shopsys/shopsys/pull/3629
+- updated tailwind to v4.1 by @chlebektomas in https://github.com/shopsys/shopsys/pull/3926
+- PersonalDataPage API contents are not mandatory by @malyMiso in https://github.com/shopsys/shopsys/pull/3882
+- storefront - upgrade sentry to 9.12 and next to 15.2 by @heca-frantisek in https://github.com/shopsys/shopsys/pull/3914
+- optimize image resizer with caching by @henzigo in https://github.com/shopsys/shopsys/pull/3907
+- Refactor of Image sizes prop by @JanMolcik in https://github.com/shopsys/shopsys/pull/3938
+- yaml standards: ignore kubernetes config files by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/3964
+- access control and admin menu tweaks by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/3933
+- dev dependencies fixes by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/3976
+- robots.txt: tweak the Crawl-delay value for new projects by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/4015
+- truly available payments for a change after the order is made are now returned from the backend by @grossmannmartin in https://github.com/shopsys/shopsys/pull/3922
+- maker bundle: add missing shopsys dependencies by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/4027
+- Improved InlineEdit and Grid administrator rights by @TomasLudvik in https://github.com/shopsys/shopsys/pull/4022
+- Add unit tests for order process util functions by @JanMolcik in https://github.com/shopsys/shopsys/pull/4060
+- enchantements for RequireOverrideAttributeSniff by @henzigo in https://github.com/shopsys/shopsys/pull/4068
+- [SSP-3360] Add Sentry Replays and Feedback with conditional lazy loading by @heca-frantisek in https://github.com/shopsys/shopsys/pull/4038
+- Implement Context system by @henzigo in https://github.com/shopsys/shopsys/pull/4064
+- updated biome to 2.2.0 and fixed issues by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4143
+- upgrade rabbitMQ version by @henzigo in https://github.com/shopsys/shopsys/pull/4167
```

Comment:
All these should be iterated and added appropriate label so they are added to correct sections here, right?